### PR TITLE
Prepare for symfony 3.4

### PR DESCRIPTION
**Pull request by Jan0707**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,6 +69,8 @@ matrix:
     - env: FOS_USER=dev-master@dev
     - env: SONATA_CORE=dev-master@dev
     - env: SONATA_ADMIN=dev-master@dev
+    - php: 7.1
+      env: SYMFONY=3.4.x-dev
```
Comment (greg0ire) [inline .travis.yml]: We manage this file with https://github.com/sonata-project/dev-kit, you should not edit it.

Comment (Jan0707) [reply .travis.yml]: Is there documentation ?

Comment (greg0ire) [reply .travis.yml]: Regarding how dev-kit works, I'm afraid not, but you don't need to touch that file to get tests for sf4. Only for sf 3.4. Here is the file we will change when 3.4 is out: https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml

**Comment by core23** [comment]
Fixed with https://github.com/sonata-project/dev-kit/pull/351

**Pull request closed without merging**